task/clair-scan/0.1/clair-scan.yaml

```diff
@@ -90,6 +103,11 @@ spec:
         # add the image_index to the processed digests list and store the result in a file
         images_processed=$(echo "${images_processed_template/\[%s]/[$digests_processed_string]}")
         echo "$images_processed" > /tekton/home/images-processed.json
+      volumeMounts:
+      - name: trusted-ca
+        mountPath: /etc/pki/tls/certs/ca-custom-bundle.crt
+        subPath: ca-bundle.crt
+        readOnly: true
     - name: conftest-vulnerabilities
```

Comment:
Is there a specific reason not to add it to the `conftest-vulnerabilities` step? If not, it may be better to put the volumeMounts in the stepTemplate to make sure it applies to all steps

Reply:
I have no objections to add it that way. I'll give it a try.

Reply:
done